modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+The following condition types are consistent across all ClusterVersion and ClusterOperator conditions. Individual components may also have additional condition types with component-specific language.
+
+The condition type, as well as additional information about your operator, can be retrieved in either YAML or JSON format through the `oc get clusterversion -o` command:
+
+[source,terminal]
+----
+$ oc get clusterversion -o yaml
+----
+
+
+* Available: 
+The condition with type `Available` indicates that the operators are functional and available in the cluster. If the status is `False`, that indicates that at least one part of the operand is non-functional and the condition requires an administrator to intervene.
+
+* Progressing:
+The condition with type `Progressing` indicates that the operator is actively rolling out new code, propagating configuration changes, or otherwise moving from one steady state to another. 
++
+Operators do not report as `Progressing` when they are reconciling a previous known state. If the observed cluster state has changed and the operator is reacting to it, then the status will report back as `True`, since it is moving from one steady state to another.
++
+* Degraded:
+The condition with type `Degraded` indicates that the operator's current state does not match its desired state over a period of time. The period of time may vary by component, but a `Degraded` state represents persistent observation of a condition.  As a result, a component will not fluctuate in and out of the `Degraded` state.  
++
+There may be a different condition type if the transition from one state to another does not persist over a long enough period to report `Degraded`.  
+An operator will not report `Degraded` during the course of a normal upgrade.  An operator may report `Degraded` in response to a persistent infrastructure failure that requires eventual administrator intervention.
++
+[NOTE]
+====
+This condition type can seem alarming, but it is simply an indication that that something may need investigation and fixing. As long as the component is available, the `Degraded` condition does not cause user workload failure or application downtime.
+====
++
+* Upgradeable:
+A condition with type `Upgradeable` indicates whether the operator is safe to upgrade based on the current cluster state. When the status is `False`, the cluster-version operator will prevent the cluster from performing impacted updates unless forced.
++
+When set on ClusterVersion, the message will explain which updates (minor or patch) are impacted. When set on ClusterOperator, the `False` status will block minor (4.y to 4.y+1) OpenShift updates. The message field will contain a human-readable description of what the administrator needs to do for the cluster or operator to successfully update. The cluster-version operator will allow updates when this condition is not `False`, including when it is `True`, `Unknown` or missing.
```

Comment:
This paragraph is confusing as it is not a general knowledge the difference between cluster version and cluster operator.

Reply:
`When set on ClusterVersion, the message will explain which updates (minor or patch) are impacted`  gives the impression that it is usual for ugradeable=false to block patch lavel upgrades. Which is not correct and can mislead the reader.